swap/src/protocol/alice/swap.rs

```diff
@@ -110,9 +114,9 @@ pub async fn run_until(
                 };
 
                 let db_state = (&state).into();
-                db.insert_latest_state(swap_id, Swap::Alice(db_state))
+                db.insert_latest_state(swap_id, database::Swap::Alice(db_state))
                     .await?;
-                run_until(
+                run_until_internal(
```

Comment:
Is this massive rename necessary? I don't understand what is the added value.

Reply:
there are two `run_until` functions now. One is exposed and the other one isn't. The one exposed is:

```
pub async fn run_until(
    swap: alice::Swap,
    is_target_state: fn(&AliceState) -> bool,
)
```

it acts as a wrapper for the internal one is: 

```
async fn run_until_internal(
    state: AliceState,
    is_target_state: fn(&AliceState) -> bool,
    mut event_loop_handle: EventLoopHandle,
    bitcoin_wallet: Arc<bitcoin::Wallet>,
    monero_wallet: Arc<monero::Wallet>,
    config: Config,
    swap_id: Uuid,
    db: Database,
)
```

@rishflab and @thomaseizinger did not like the name `do_run_until` for the internal one so I changed it to `run_until_internal`. I don't think it is very important to agree on the perfect name for it. Happy for follow up PRs on that one :)